tembo-operator/src/defaults.rs

```diff
@@ -43,12 +43,38 @@ pub fn default_port() -> i32 {
     5432
 }
 
+pub fn default_repository() -> String {
+    "quay.io/tembo".to_owned()
+}
+
 pub fn default_image() -> String {
-    "quay.io/tembo/standard-cnpg:15.3.0-1-0c19c7e".to_owned()
+    "standard-cnpg:15.3.0-1-0c19c7e".to_owned()
 }
 
 pub fn default_llm_image() -> String {
-    "quay.io/tembo/ml-cnpg:15.3.0-1-63e32a1".to_owned()
+    "ml-cnpg:15.3.0-1-63e32a1".to_owned()
```

Comment:
Is it possible for the values in this file to come from the stack yamls? So it's like if someone updates the image in a stack yaml, then the operator tests run in CI and it's using the new image from the stack yaml for which there are operator tests?

If we don't do that, then our operator tests use the image configured here in the defaults, but a future change to stack yaml, then following that to update in control plane, can result in us use image defaults that have not been tested by the functional testing in the operator code